projects/batfish/src/test/java/org/batfish/job/BatfishJobExecutorTest.java

```diff
@@ -0,0 +1,218 @@
+package org.batfish.job;
+
+import static org.hamcrest.Matchers.lessThanOrEqualTo;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import org.batfish.common.BatfishException;
+import org.batfish.common.BatfishLogger;
+import org.batfish.common.BatfishLogger.BatfishLoggerHistory;
+import org.batfish.common.CompositeBatfishException;
+import org.batfish.common.util.CommonUtil;
+import org.batfish.config.Settings;
+import org.batfish.datamodel.answers.AnswerElement;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+/** Tests for {@link BatfishJobExecutor}. */
+public class BatfishJobExecutorTest {
+
+  @Rule public ExpectedException _thrown = ExpectedException.none();
+  BatfishLogger _logger;
+
+  @Before
+  public void setup() {
+    _logger = new BatfishLogger("info", false);
```

Comment:
Isn't there a `BfConsts` constant you can use here instead of `"info"`?

Reply:
done